src/ErrorBoundary.ts

```diff
@@ -29,6 +36,9 @@ export class ErrorBoundary extends Component<
     this.state = initialState;
   }
 
+  static defaultProps = {
+    suppressLogging: false,
```

Comment:
I'm not really sure what this prop is supposed to do. The behavioral change we originally discussed was to auto-suppress errors boundaries triggered by `useErrorBoundary`. That seems like a separate concern from an `ErrorBoundary` instance.

Reply:
I apologize for any confusion. The goal remains to auto-suppress the console error message triggered by `useErrorBoundary`. Since this is an additional feature, I think it's better not to roll it out to existing users unless they actively choose to enable it. That’s why I introduced the `suppressLogging` property, allowing users to opt into this new feature when they’re ready.